src/screens/OtherProfile/index.js

```diff
@@ -87,20 +89,23 @@ const OtherProfile = () => {
   const interactionManagerRef = React.useRef(null);
   const [otherProfileFeeds, dispatchOtherProfile] = React.useContext(Context).otherProfileFeed;
   const [profile] = React.useContext(Context).profile;
-  const [, dispatch] = React.useContext(Context).feeds;
-  const [isLastPage, setIsLastPage] = React.useState(false);
+  const [mainFeeds, dispatch] = React.useContext(Context).feeds;
   const [loading, setLoading] = React.useState(false);
   const [initLoading, setInitLoading] = React.useState(true);
   const [isAnonimity, setIsAnonimity] = React.useState(false);
   const {params} = route;
   const {feeds} = otherProfileFeeds;
+  const {timer, viewPostTimeIndex} = mainFeeds;
   const [loadingGenerateAnon, setLoadingGenerateAnon] = React.useState(false);
   const [anonProfile, setAnonProfile] = React.useState();
   const {mappingColorFeed} = useCoreFeed();
   const isSignedMessageEnabled = dataMain.isSignedMessageEnabled ?? true;
   const isAnonimityEnabled = dataMain.isAnonMessageEnabled && isSignedMessageEnabled;
   const {createSignChat} = useCreateChat();
 
+  const {onWillSendViewPostTime} = useViewPostTimeHook(dispatch, timer, viewPostTimeIndex);
+  const {fetchNextFeeds} = useFeedPreloadHook(feeds?.length, () => getOtherFeeds(postOffset));
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code introduces a few changes. The `isLastPage` state variable has been removed, which was previously used to determine if the end of the feed had been reached. This logic seems to have been replaced by the `fetchNextFeeds` function from the `useFeedPreloadHook`. 

However, it's not clear how the `postOffset` is being updated in the new code. In the old code, `getOtherFeeds(postOffset)` would fetch feeds based on the current offset. If `postOffset` isn't properly updated, this could lead to issues with fetching and displaying the correct feeds.

Please ensure that the `postOffset` is being correctly updated when fetching the next set of feeds.



<!-- This is an auto-generated comment by OSS CodeRabbit -->